demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -321,10 +401,14 @@ def _find_pack_dependencies(pack_id, id_set):
         Returns:
             set: dependencies data that includes pack id and whether is mandatory or not.
         """
-        pack_scripts, pack_playbooks = PackDependencies._collect_pack_items(pack_id, id_set)
-        scripts_dependencies = PackDependencies._collect_scripts_dependencies(pack_scripts, id_set)
-        playbooks_dependencies = PackDependencies._collect_playbooks_dependencies(pack_playbooks, id_set)
-        pack_dependencies = scripts_dependencies | playbooks_dependencies
+        pack_items = PackDependencies._collect_pack_items(pack_id, id_set)
+        scripts_dependencies = PackDependencies._collect_scripts_dependencies(pack_items['scripts'], id_set)
+        playbooks_dependencies = PackDependencies._collect_playbooks_dependencies(pack_items['playbooks'], id_set)
+        layouts_dependencies = PackDependencies._collect_layouts_dependencies(pack_items['layouts'], id_set)
+        incidents_fields_dependencies = PackDependencies. \
+            _collect_incidents_fields_dependencies(pack_items['incidents_fields'], id_set)
+        pack_dependencies = scripts_dependencies | playbooks_dependencies | layouts_dependencies | \
+            incidents_fields_dependencies
         # todo check if need to collect dependencies from other content items
 
```

Comment:
https://github.com/demisto/demisto-sdk/blob/333b2e865e9888d2b35c8751311dbfa09a775bbc/demisto_sdk/tests/test_files/id_set/id_set.json is real id-set that was generated according to your changes, correct?